modules/services/cliphist.nix

```diff
@@ -17,6 +17,14 @@ in {
       '';
     };
 
+    flags = lib.mkOption {
```

Comment:
This type of option is typically called `extraOptions`. If you search, you'll find multiple examples so I would suggest to follow that convention:
```suggestion
    extraOptions = lib.mkOption {
```

Reply:
Huh, I didn't know that yet, and remembered adding the flag option to atuin some time ago. Thanks for sharing.

Reply:
No worries, it's not a hard rule or anything.